src/Metalama/Metalama.Compiler.Sdk/build/Metalama.Compiler.Sdk.targets

```diff
@@ -96,18 +96,19 @@
   </Target>
 
   <Target Name="MetalamaCompilerUpdateNuspec" DependsOnTargets="MetalamaCompilerGenerateNuspec" Condition="$(IsPackable) == 'true'  AND $(MetalamaCompilerDisablePackCustomization)!='true'">
+
     <ItemGroup>
-      <FilteredOutput Include="@(NuGetPackOutput)" Condition="%(Extension) == '.nuspec'"/>
+      <NuspecFileAbsolutePaths Include="@(NuGetPackOutput)" Condition="%(Extension) == '.nuspec'"/>
     </ItemGroup>
 
-    <PropertyGroup>
-      <NuspecFileAbsolutePath>@(FilteredOutput)</NuspecFileAbsolutePath>
-    </PropertyGroup>
     <ItemGroup>
       <MetalamaSdkPackageReplacement Include="Metalama.Compiler.Sdk" ReplacedBy="Metalama.Compiler"/>
     </ItemGroup>
 
-    <RewriteMetalamaSdkNuspec File="$(NuspecFileAbsolutePath)" Replacements="@(MetalamaSdkPackageReplacement)"/>
+    <Message Text="NuspecFileAbsolutePaths.Identity: %(NuspecFileAbsolutePaths.Identity)" Importance="High" />
+    <Message Text="MetalamaSdkPackageReplacement: @(MetalamaSdkPackageReplacement)" Importance="High" />
```

Comment:
These logging messages probably shouldn't be kept, or at least not with `Importance="High"`.